microsetta_interface/templates/submit_interest.jinja2

```diff
@@ -63,7 +67,17 @@ $(document).ready(function(){
                     required: true,
                     phoneSPAIN: true
                 });
-
+                $("#city").rules( "add", {
+                    required: true
+                });
+                $("#state").rules( "add", {
+                    required: true
+                });
+            } else if (selected_country == "JP") {
+                $("#form_group_address_3").show();
```

Comment:
Just to check, have example Japanese addresses been attempted in this interface? I know they differ somewhat from the structure used in the US. I suspect the third line is good enough, but I'm not positive. Related, will this third address line be exposed to Melissa for validation?

Reply:
Sagawa provided the format and I've tested that the form collects what they're looking for. I'll have Promi/Se Jin consult with them to make sure the instructions direct users to enter the right values in the right places.

As far as Melissa validation, in pre-Spain testing, we were seeing results that conflicted with what we knew to be valid or invalid addresses, so we decided to disable the address verification for the time being. I did plug Address 3 into the Melissa code before we made that decision, so the plumbing is there if/when we decided to flip it back on.

Reply:
Okay great, thanks